### PR TITLE
Complete sign up process

### PR DESCRIPTION
**Pull request by fjvallarino**
@ezequielc please review

### CODE REVIEW

File: src/userstate/auth-form.html [hunk 1]
```diff
@@ -38,17 +38,20 @@
...
     </div>
 
-    <div class="form-group" ng-class="{'has-error': (forms.loginForm.$submitted && forms.loginForm.password.$invalid)}" show-errors>
+    <div class="form-group" ng-class="{'has-error': (forms.loginForm.$submitted && !isPasswordValid() && isSignUp), 'has-message': isPasswordValid() && isSignUp}" show-errors>
       <label class="control-label">Password</label>
       <input type="password" class="form-control" placeholder="Enter Password" id="password" name="password" ng-model="credentials.password" required>
-      <p class="text-danger" ng-show="forms.loginForm.$submitted && forms.loginForm.password.$invalid">
-        Please enter a Password
+      <p class="text-danger" ng-show="forms.loginForm.$submitted && !isPasswordValid() && isSignUp">
+        Please enter a valid Password
```
Comment (ezequielc) [inline src/userstate/auth-form.html]: @fjvallarino Could you change this message to 'Please enter at least 4 characters.'.

**Comment by ezequielc** [comment]
@fjvallarino Code looks good. Thanks

**Comment by fjvallarino** [comment]
@ezequielc regarding the issues:

1. I left the confirmation dialog in order to be consistent with the way the Google accounts handle logout (it was probably easier to just sign out from the main link). Maybe we can discuss during retrospection.
2. Deployed on stage-3
3. Deployed on stage-3
4. Deployed on stage-3

I'll check the other items now. Password change styling has been updated to reflect the prototype. Animations are not being applied, since they were using jQuery. I think we'll have to follow ng-animate css rules in order to have the effects applied on Apps.

**Comment by fjvallarino** [comment]
@ezequielc please review

**Comment by ezequielc** [comment]
@fjvallarino 👍

**Pull request merged**